src/main/java/org/hl7/davinci/priorauth/App.java

```diff
@@ -52,16 +55,9 @@ public static void main(String[] args) {
     initializeAppDB();
 
     // Assemble the microservice
-    Meecrowave.Builder builder = new Meecrowave.Builder();
-    builder.setHttpPort(9000);
-    builder.setScanningPackageIncludes("org.hl7.davinci.priorauth");
-    builder.setJaxrsMapping("/fhir/*");
-    builder.setJsonpPrettify(true);
-
-    // Launch the microservice
-    try (Meecrowave meecrowave = new Meecrowave(builder)) {
-      meecrowave.bake().await();
-    }
+    SpringApplication server = new SpringApplication(App.class);
+    server.setDefaultProperties(Collections.singletonMap("server.port", "9000"));
```

Comment:
Does is still use /fhir in the path for the endpoint?

Reply:
Yes, but not by default. I made PRIORAUTH-40 to look into setting the context path to be `/fhir`. I tried doing this but the tests started failing so I need to look into how that works. For the time being each controller has been set to `/fhir/name`

Reply:
Did some research into contextPath for tests and found a solution. Updated this branch with the new tests